docs/notebooks/datasets/simlex-999.txt

```diff
@@ -0,0 +1,999 @@
+old	new	1.58
```

Comment:
Please move to `test/test_data` so it can be used in other code. Similar to https://github.com/parulsethi/gensim/blob/develop/gensim/test/test_data/wordsim353.tsv